static/app/components/tabs/index.tsx

```diff
@@ -73,6 +77,7 @@ const TabsWrap = styled('div', {shouldForwardProp: tabsShouldForwardProp})<{
 }>`
   display: flex;
   flex-direction: ${p => (p.orientation === 'horizontal' ? 'column' : 'row')};
+  flex-grow: 1;
```

Comment:
Just gonna slip this in… If this is a flex item (has a flex parent), we almost certainly want it to grow as large as it can. Adding this as a default style removes the need to restyle the component.